src/main/java/org/rumbledb/compiler/StaticContextVisitor.java

```diff
@@ -130,7 +130,7 @@ public StaticContext visitFunctionDeclaration(InlineFunctionExpression expressio
             .forEach(
                 (paramName, flworVarSequenceType) -> functionDeclarationContext.addVariable(
                     paramName,
-                    flworVarSequenceType.getSequence(),
+                    flworVarSequenceType,
```

Comment:
It would be great if we could rename this variable. It looks out of place as is.

Reply:
DONE